package-lock.json

```diff
@@ -23831,7 +23831,7 @@
       "dependencies": {
         "async": {
           "version": "1.5.2",
-          "resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
+          "resolved": "http://registry.npmjs.org/async/-/async-1.5.2.tgz",
```

Comment:
What version of `npm` are you running? Can you update `package.json:engines.npm` to match the most recent version?

Reply:
Running npm version 6.9.0.

I'll update it for sure.